VogtObserver/Generator.cs

```diff
@@ -4,7 +4,34 @@
 
 namespace VogtObserver
 {
-    class Generator
+    class Generator : IObservable
```

Comment:
Ok so I thin kthe class names were throwing me off for a while.

This generator is what is observable. But what is it generating that other things observe? A random number.

I'd call this something more like.. RandomNumberGenerator.